stencil-library/readme.md

```diff
@@ -1,15 +1,41 @@
-## Usage
+<p align="center">
+  <a href="https://justifi.tech/" rel="noopener" target="_blank"><img width="150" height="133" src="https://justifi.tech/wp-content/uploads/2022/07/Light-BG-1.svg" alt="JustiFi logo"></a>
+</p>
 
-There are two methods for including the JustiFi web components in your application:
+<h1 align="center">JustiFi Web Components</h1>
 
-### Installation via npm
+The JustiFi web component library is our collection of customizable fintech components. The library includes various types of components such as Payment Forms, Tables/Lists, Charts, Onboarding Forms, and more. It is built for our clients to simply and efficiently implement key pieces of their finance infrastructure into their existing dashboards or software solutions.
```

Comment:
@jakemer @TiagoFuelber what do you guys think of this section? Any improvements or ideas are welcome!

Reply:
Nice, I think it's clear. IMO